### DISCUSSION TITLE
Schedule Qfield Sync to run every 15 minutes.

### DISCUSSION THREAD

**Discussion by troussell** [opengisch/QField#3067] [Q&A]
Is it possible to schedule Qfield Sync to run every 15 minutes on our server to check for updates on the cloud and apply any changes to our database?

**Comment by lucienicolier** [comment]
Hi,

You have three possibility.

1. Connect your database directly with QFieldCloud. It has to be a Postgres database and visible on internet.
2. You can use SDK, see [SDK](https://docs.qfield.org/reference/qfieldcloud/sdk/)
3. We can make you an offer to help you and customize the solution

I hope it helps
